tests/e2e/util/common/e2e_utils.go

```diff
@@ -90,28 +92,57 @@ func CheckNamespaceEmpty(ctx SpecContext, cl client.Client, ns string) {
 
 func LogDebugInfo() {
 	// General debugging information to help diagnose the failure
-	// TODO: Add more debugging information for others resources
-	resource, err := kubectl.GetYAML(controlPlaneNamespace, "istio", istioName)
+	// TODO: Add the creation of file with this information to be attached to the test report
+
+	GinkgoWriter.Println("The test run has failures and the debug information is as follows:")
+	// Display Operator information
+	operator, err := kubectl.GetYAML(namespace, "deployment", deploymentName)
 	if err != nil {
-		GinkgoWriter.Println("Error getting Istio CR: ", err)
+		GinkgoWriter.Println("Error getting operator deployment yaml: ", err)
 	}
-	GinkgoWriter.Println("Istio CR: \n", resource)
+	GinkgoWriter.Println("Operator deployment: \n", operator)
 
-	output, err := kubectl.GetPods(controlPlaneNamespace, "-o wide")
+	describe, err := kubectl.Describe(namespace, "deployment", deploymentName)
```

Comment:
Why do we need both the YAML and the description? They both show the same information. I'd remove the description and leave just the YAML.